.github/workflows/automatic-doc-checks.yml

```
@@ -0,0 +1,16 @@
+name: Main Documentation Checks
+
+on:
+  - push
+  - pull_request
+  - workflow_dispatch
```

I think just manual for now?

```suggestion
  - workflow_dispatch
```